### PR TITLE
Am 2611 specific challenged validation drools rules judiciary

### PR DESCRIPTION
**Pull request by mikebrownccd**
https://tools.hmcts.net/jira/browse/AM-2611
ET_WA - Enable Specific and Challenged access for ET for both STAFF and JUDICIARY

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/resources/validationrules/employment/common/employment-challenged-access.drl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import uk.gov.hmcts.reform.roleassignment.domain.model.CaseAllocatorApproval;
+
+/*
+ * EMPLOYMENT "challenged-access-judiciary" case validation rule.
+ */
+rule "employment_challenged_access_judiciary_create_case_roles"
+when
+    $ca: CaseAllocatorApproval(
```
Comment (sankar-padakula) [inline src/main/resources/validationrules/employment/common/employment-challenged-access.drl:26]: this rule is not required. they have two casetypes where both has to allow. So no need specific rule for that. generic rule will work.

**Review by sankar-padakula — APPROVED** [review]

File: src/main/resources/validationrules/employment/common/employment-challenged-access.drl (hunk 1 above)
Comment (mikebrownccd) [reply src/main/resources/validationrules/employment/common/employment-challenged-access.drl]: these files have been removed from the PR and the tests continue to pass, thanks for the advice

File: src/main/resources/validationrules/employment/common/employment-specific-access.drl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import uk.gov.hmcts.reform.roleassignment.domain.model.enums.FeatureFlagEnum;
+import uk.gov.hmcts.reform.roleassignment.util.ValidationUtil;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import uk.gov.hmcts.reform.roleassignment.domain.model.CaseAllocatorApproval;
+
+/*
+ * EMPLOYMENT "specific-access-judiciary" case validation rule.
+ */
+rule "employment_specific_access_judiciary_create_case_roles"
```
[the remark this answers is not in the document]
Comment (mikebrownccd) [reply src/main/resources/validationrules/employment/common/employment-specific-access.drl]: these files have been removed from the PR and the tests continue to pass, thanks for the advice

**Comment by mattnayler** [comment]
Reviewed alongside #1863 (+ #1889) and current documented mappings in config; see comments against #1863.

**Pull request merged**